### ISSUE TITLE
INSTALLATION: DeprecationWarning: pkg_resources is deprecated in kilosort/__init__.py

### ISSUE THREAD

**Issue by cyf203** [MouseLand/Kilosort#800]
### Describe the issue:

After I installed Kilosort without GUI in Python 3.10 environment, I found the deprecation warning as shown in the title. I found this might be a potential problem. 

To solve this, I did:
`from pkg_resources import get_distribution, DistributionNotFound`
`__version__ = version(__name__)`
`except DistributionNotFound:`

Since this is such a small issue, I didn't fork the repository. 

### Provide environment info:

channels:
  - pytorch
  - nvidia
  - conda-forge
  - defaults
dependencies:
  - asttokens=2.4.1=pyhd8ed1ab_0
  - blas=1.0=mkl
  - bzip2=1.0.8=h2bbff1b_6
  - ca-certificates=2024.9.24=haa95532_0
  - colorama=0.4.6=pyhd8ed1ab_0
  - comm=0.2.2=pyhd8ed1ab_0
  - cpython=3.10.15=py310hd8ed1ab_1
  - cuda-cccl=12.6.77=0
  - cuda-cccl_win-64=12.6.77=0
  - cuda-cudart=12.4.127=0
  - cuda-cudart-dev=12.4.127=0
  - cuda-cupti=12.4.127=0
  - cuda-libraries=12.4.0=0
  - cuda-libraries-dev=12.4.0=0
  - cuda-nvrtc=12.4.127=0
  - cuda-nvrtc-dev=12.4.127=0
  - cuda-nvtx=12.4.127=0
  - cuda-opencl=12.6.77=0
  - cuda-opencl-dev=12.6.77=0
  - cuda-profiler-api=12.6.77=0
  - cuda-runtime=12.4.0=0
  - cuda-version=12.6=3
  - debugpy=1.8.6=py310h9e98ed7_0
  - decorator=5.1.1=pyhd8ed1ab_0
  - exceptiongroup=1.2.2=pyhd8ed1ab_0
  - executing=2.1.0=pyhd8ed1ab_0
  - gmpy2=2.1.2=py310h7f96b67_0
  - importlib-metadata=8.5.0=pyha770c72_0
  - intel-openmp=2023.1.0=h59b6b97_46320
  - ipykernel=6.29.5=pyh4bbf305_0
  - ipython=8.28.0=pyh7428d3b_0
  - jedi=0.19.1=pyhd8ed1ab_0
  - jinja2=3.1.4=py310haa95532_0
  - jupyter_client=8.6.3=pyhd8ed1ab_0
  - jupyter_core=5.7.2=pyh5737063_1
  - krb5=1.21.3=hdf4eb48_0
  - libcublas=12.4.2.65=0
  - libcublas-dev=12.4.2.65=0
  - libcufft=11.2.0.44=0
  - libcufft-dev=11.2.0.44=0
  - libcurand=10.3.7.77=0
  - libcurand-dev=10.3.7.77=0
  - libcusolver=11.6.0.99=0
  - libcusolver-dev=11.6.0.99=0
  - libcusparse=12.3.0.142=0
  - libcusparse-dev=12.3.0.142=0
  - libffi=3.4.4=hd77b12b_1
  - libnpp=12.2.5.2=0
  - libnpp-dev=12.2.5.2=0
  - libnvfatbin=12.6.77=0
  - libnvfatbin-dev=12.6.77=0
  - libnvjitlink=12.4.99=0
  - libnvjitlink-dev=12.4.99=0
  - libnvjpeg=12.3.1.89=0
  - libnvjpeg-dev=12.3.1.89=0
  - libsodium=1.0.20=hc70643c_0
  - libuv=1.48.0=h827c3e9_0
  - matplotlib-inline=0.1.7=pyhd8ed1ab_0
  - mkl=2023.1.0=h6b88ed4_46358
  - mpc=1.1.0=h7edee0f_1
  - mpfr=4.0.2=h62dcd97_1
  - mpir=3.0.0=hec2e145_1
  - mpmath=1.3.0=py310haa95532_0
  - nest-asyncio=1.6.0=pyhd8ed1ab_0
  - openssl=3.3.2=h2466b09_0
  - packaging=24.1=pyhd8ed1ab_0
  - parso=0.8.4=pyhd8ed1ab_0
  - pickleshare=0.7.5=py_1003
  - pip=24.2=py310haa95532_0
  - platformdirs=4.3.6=pyhd8ed1ab_0
  - prompt-toolkit=3.0.48=pyha770c72_0
  - psutil=6.0.0=py310ha8f682b_1
  - pure_eval=0.2.3=pyhd8ed1ab_0
  - pygments=2.18.0=pyhd8ed1ab_0
  - python=3.10.15=h4607a30_1
  - python_abi=3.10=2_cp310
  - pytorch=2.4.1=py3.10_cuda12.4_cudnn9_0
  - pytorch-cuda=12.4=h3fd98bf_6
  - pytorch-mutex=1.0=cuda
  - pyzmq=26.2.0=py310h656833d_2
  - setuptools=75.1.0=py310haa95532_0
  - six=1.16.0=pyh6c4a22f_0
  - sqlite=3.45.3=h2bbff1b_0
  - stack_data=0.6.2=pyhd8ed1ab_0
  - tbb=2021.8.0=h59b6b97_0
  - tk=8.6.14=h0416ee5_0
  - tornado=6.4.1=py310ha8f682b_1
  - traitlets=5.14.3=pyhd8ed1ab_0
  - typing_extensions=4.12.2=pyha770c72_0
  - ucrt=10.0.22621.0=h57928b3_1
  - vc=14.40=h2eaa2aa_1
  - vc14_runtime=14.40.33810=hcc2c482_22
  - vs2015_runtime=14.40.33810=h3bf8584_22
  - wcwidth=0.2.13=pyhd8ed1ab_0
  - wheel=0.44.0=py310haa95532_0
  - xz=5.4.6=h8cc25b3_1
  - yaml=0.2.5=he774522_0
  - zeromq=4.3.5=ha9f60a1_6
  - zipp=3.20.2=pyhd8ed1ab_0
  - zlib=1.2.13=h8cc25b3_1
  - pip:
      - aiohappyeyeballs==2.4.3
      - aiohttp==3.10.9
      - aiosignal==1.3.1
      - altair==5.4.1
      - asciitree==0.3.3
      - async-timeout==4.0.3
      - attrs==24.2.0
      - cbor2==5.6.5
      - certifi==2024.8.30
      - cffi==1.17.1
      - charset-normalizer==3.4.0
      - click==8.1.7
      - cloudpickle==3.0.0
      - contourpy==1.3.0
      - cryptography==43.0.1
      - cuda-python==12.6.0
      - cycler==0.12.1
      - dask==2024.9.1
      - distinctipy==1.3.4
      - distributed==2024.9.1
      - faiss-cpu==1.9.0
      - fasteners==0.19
      - figurl==0.2.22
      - filelock==3.16.1
      - fonttools==4.54.1
      - frozenlist==1.4.1
      - fsspec==2024.9.0
      - h5py==3.12.1
      - hdmf==3.14.5
      - idna==3.10
      - ipympl==0.9.4
      - ipython-genutils==0.2.0
      - ipywidgets==8.1.5
      - isosplit5==0.2.2
      - isosplit6==0.1.4
      - joblib==1.4.2
      - jsonschema==4.23.0
      - jsonschema-specifications==2024.10.1
      - jupyterlab-widgets==3.0.13
      - kachery-cloud==0.4.9
      - kilosort==4.0.18
      - kiwisolver==1.4.7
      - llvmlite==0.43.0
      - locket==1.0.0
      - markupsafe==3.0.1
      - matplotlib==3.9.2
      - mountainsort4==1.0.6
      - mountainsort5==0.5.6
      - msgpack==1.1.0
      - multidict==6.1.0
      - narwhals==1.9.2
      - neo==0.13.3
      - networkx==3.4
      - numba==0.60.0
      - numcodecs==0.13.1
      - numpy==1.26.4
      - pandas==2.2.3
      - partd==1.4.2
      - pillow==10.4.0
      - probeinterface==0.2.24
      - propcache==0.2.0
      - pubnub==9.0.0
      - pybind11==2.13.6
      - pycparser==2.22
      - pycryptodomex==3.21.0
      - pynwb==2.8.2
      - pyparsing==3.1.4
      - pyqtgraph==0.13.7
      - python-dateutil==2.9.0.post0
      - pytz==2024.2
      - pywin32==307
      - pyyaml==6.0.2
      - quantities==0.15.0
      - referencing==0.35.1
      - requests==2.32.3
      - rpds-py==0.20.0
      - ruamel-yaml==0.18.6
      - ruamel-yaml-clib==0.2.8
      - scikit-learn==1.5.2
      - scipy==1.14.1
      - simplejson==3.19.3
      - sortedcontainers==2.4.0
      - sortingview==0.13.5
      - spikeextractors==0.9.11
      - spikeinterface==0.101.2
      - spikeinterface-gui==0.9.1
      - stack-data==0.6.3
      - sympy==1.13.3
      - tblib==3.0.0
      - threadpoolctl==3.5.0
      - toolz==1.0.0
      - tqdm==4.66.5
      - tzdata==2024.2
      - urllib3==2.2.3
      - widgetsnbextension==4.0.13
      - yarl==1.14.0
      - zarr==2.17.2
      - zict==3.0.0

### Operating system information:

Edition	Windows 10 Enterprise
Version	22H2
Installed on	‎9/‎2/‎2020
OS build	19045.4894
Experience	Windows Feature Experience Pack 1000.19060.1000.0

**Comment by jacobpennington** [comment]
Thanks, this is fixed with the most recent commit.
